test_requirements.txt

```diff
@@ -1,7 +1,7 @@
 wheel==0.36.2
 
 # Test requirements
-pytest>=6.0.1,<6.2.2
+pytest>=6.0.1,<6.2.3
```

Comment:
```suggestion
pytest==6.2.2
```

Reply:
Wait why pin to `6.2.2` rather than keep as a range?